indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -236,6 +236,16 @@
                                     {% endif %}
                                 </td>
                             </tr>
+                            {% for item in static_columns %}
+                                {% if item.id == 'description' %}
```

Comment:
Since it's unlikely that we need to handle other columns here, maybe we could use this shortcut?

```
{% for item in static_columns if item.id == 'description' %}
```